src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -37,7 +37,8 @@ public class JsonUtil {
             .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
             .registerModule(new SimpleModule("SimpleModule")
                     .addSerializer(Level.class, new ToStringSerializer())
-                    .addDeserializer(Level.class, new LevelDeserializer(Level.class)));
+                    .addDeserializer(Level.class, new LevelDeserializer(Level.class))
+                    .addSerializer(Path.class, new ToStringSerializer()));
```

Comment:
Would there be a need to deserialise the `Path.class` as well?
```suggestion
                    .addSerializer(Path.class, new ToStringSerializer())
                    .addDeserializer(Path.class, new PathDeserializer(Path.class)));
```

Reply:
By default, Jackson will use `com.fasterxml.jackson.databind.ext.NioPathDeserializer` to deserialize the `Path` class, which we should not override as it is working as expected for both absolute and relative paths.